packages/store/addon/-private/system/core-store.ts

```diff
@@ -856,6 +855,7 @@ abstract class CoreStore extends Service {
     // app/adapters/post.js
     import ApplicationAdapter from "./application";
 
+    export default class PostAdapter extends ApplicationAdapter {
     export default ApplicationAdapter.extend({
```

Comment:
✂️ ?

Reply:
Not sure what needs to be cut/fixed here?

Reply:
Looks like the class is duplicated...So ✂️ `export default ApplicationAdapter.extend({`

Reply:
I stared at that for five whole  minutes and just couldn't see it :(